### PR TITLE
Fix issue allow mocking store module without getters

### PR DESCRIPTION
**Pull request by sarngru**
Fixes Issue https://github.com/AmpleOrganics/vue-test-utils-helpers/issues/4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-test-utils-helpers",
-  "version": "1.0.3",
+  "version": "1.0.5",
```
Comment (martinlaws) [inline package.json]: Why are we skipping `1.0.4`?

File: src/mockStoreActionsAndGetters.js [hunk 2]
```diff
@@ -10,23 +10,27 @@ function mockStoreActionsAndGetters ({
...
-      const mockActionKey = isNamespaced ? `${moduleKey}/${key}` : key
-      const mockFn = jestFn().mockResolvedValue({})
-      actionsDictionary[mockActionKey] = mockFn
-      moduleValue.actions[key] = mockFn
-    })
+    if (moduleValue.actions) {
+      Object.keys(moduleValue.actions).forEach(key => {
+        const mockActionKey = isNamespaced ? `${moduleKey}/${key}` : key
+        const mockFn = jestFn().mockResolvedValue({})
+        actionsDictionary[mockActionKey] = mockFn
```
Comment (rob0t7) [inline src/mockStoreActionsAndGetters.js]: Could you rewrite this using _reduce_?

**Comment by rob0t7** [comment]
Hey Gru,

Just some general feedback...

Could you split this function into using two helper functions? By using named helper functions (ones that are not exposed outside of the module) it would improve the readablity of what the code is trying to accomplish. It would also be a very simple change that would only involve moving the code around and not changing it.

Would you be up for that? If you want help let me know?

File: package.json (hunk 1 above)
Comment (sarngru) [reply package.json]: I missed updating package.json with correct version last time (1.0.4). 

we already have a release 1.0.4 but package.json is still version. this pr syncs release and package.json

File: src/mockStoreActionsAndGetters.js (hunk 2 above)
Comment (bobmanary) [reply src/mockStoreActionsAndGetters.js]: I find that using `reduce` when the desired output is a map/dictionary object ends up being a bit awkward.

Comment (sarngru) [reply src/mockStoreActionsAndGetters.js]: I am in agreement with @bobmanary, using `reduce` in this use case will make code less readable. Here is a version using `reduce`

```javascript
Object.entries(moduleValue.actions).reduce((actionsDictionary, [key]) => {
      const mockActionKey = isNamespaced ? `${moduleKey}/${key}` : key
      const mockFn = jestFn().mockResolvedValue({})
      actionsDictionary[mockActionKey] = mockFn
      moduleValue.actions[key] = mockFn
      return actionsDictionary
    }, actionsDictionary)
```

**Comment by sarngru** [comment]
> Hey Gru,
> 
> Just some general feedback...
> 
> Could you split this function into using two helper functions? By using named helper functions (ones that are not exposed outside of the module) it would improve the readablity of what the code is trying to accomplish. It would also be a very simple change that would only involve moving the code around and not changing it.
> 
> Would you be up for that? If you want help let me know?

Fixed. Extracted two functions `mockStoreActions` and `mockStoreGetters`.

**Comment by sarngru** [comment]
I have removed changes related to version 1.0.5 from ChangeLog.md. I will submit them as another pr after this pr gets merged

**Comment by ZainW** [comment]
could you please keep version bumping and changelog, i think it will get confusing if we do patterns like this for releasing

**Comment by martinlaws** [comment]
Great work Gru, thanks for taking @rob0t7's feedback and splitting into two functions adhering to the principle of single responsibility. Super readable 👍 

Please rebase against master and we can merge

**Pull request merged**